frontend_lib/test/component/NewShareDownload.spec.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react'
+import { expect } from 'chai'
+import { shallow } from 'enzyme'
+import sinon from 'sinon'
+import { NewShareDownload } from '../../src/component/ShareDownload/NewShareDownload.jsx'
+require('../../src/component/ShareDownload/ShareDownload.styl')
+
+describe('<NewShareDownload />', () => {
+  const onChangeEmailsCallBack = sinon.stub()
+  const onKeyDownEnterCallBack = sinon.stub()
+  const onChangePasswordCallBack = sinon.stub()
+  const onClickCancelButtonCallBack = sinon.stub()
+  const onClickNewShareCallBack = sinon.stub()
+
+  const props = {
+    shareEmails: 'randomEmail@randomEmail.randomEmail',
+    onChangeEmails: onChangeEmailsCallBack,
+    onKeyDownEnter: onKeyDownEnterCallBack,
+    hexcolor: 'yellow',
+    sharePassword: 'randomPassword',
+    onChangePassword: onChangePasswordCallBack,
+    onClickCancelButton: onClickCancelButtonCallBack,
+    onClickNewShare: onClickNewShareCallBack,
+    emailNotifActivated: false
+  }
+
+  const wrapper = shallow(
+    <NewShareDownload
+      {...props}
+      t={key => key}
```

Comment:
don't add `t` props manually, use a mock HOC.

Copy this example into frontend_lib:
https://github.com/tracim/tracim/blob/develop/frontend/test/hocMock/translate.js